mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -45,6 +45,21 @@
 @SuppressWarnings("deprecation")
 public class FakeMailContext implements MailetContext {
 
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static FakeMailContext defaultContext() {
+        return builder().build();
```

Comment:
Sorry I don't see the interest of using a builder here instead of just returning new FakeMailContext() here.